docs/solidity-to-cadence.mdx

```diff
@@ -0,0 +1,447 @@
+to that account's storage, public keys and contract code. Capabilities ensure that resources held in an account can be
+safely shared/accessed; access to `AuthAccount` is never given to other accounts.
+
+## Resources
+
+Resources are unique, [linear-types](https://en.wikipedia.org/wiki/Substructural_type_system#Linear_type_systems) which
+can never be copied or implicitly discarded, only moved between accounts. If, during development, a function fails to
+store a Resource obtained from an account in the function scope, semantic checks will flag an error. The run-time
+enforces the same strict rules in terms of allowed operations. Therefore, contract functions which do not properly
+handle Resources in scope before exiting will abort, reverting them to the original storage. These features of
+Resources make them perfect for representing tokens, both fungible and non-fungible. Ownership is tracked by where
+they are stored, and the assets can’t be duplicated or accidentally lost since the language itself enforces correctness.
+
+Flow encourages storing of data and compute on-chain and Resource-types makes this easier than even. Since Resources
+are always stored in accounts, any data that exists in Resource instances is seamlessly being managed on-chain
+without any explicit handling needed.
+
+## Capability-based access
+
+Remote access to stored objects can be delegated via [Capabilities](language#capability-based-access-control). This
+means that if an account wants to be able to access another account's stored objects, it must have been provided
+with a valid Capability to that object. Capabilities can be either public or private. An account can share a public
+Capability if it wants to give all other accounts access. (For example, it’s common for an account to accept fungible
+token deposits from all sources via a public Capability.) Alternatively, an account can grant private Capabilities
+to specific accounts in order to provide access to restricted functionality. For example, an NFT project often
+controls minting through an “administrator Capability” that grants specific accounts with the power to mint new tokens.
+
+## Contract standards
+
+There are numerous widely-used contract standards established to benefit the ecosystem, for example
+[Fungible Token](flow#core-contracts#fungible-token)(FT) and [Non-Fungible Token](flow#core-contracts#non-fungible-token)(NFT)
+standards which are conceptually equivalent to Ethereum's ERC-20 and ERC-721 standards. Cadence's object-oriented
+design means standards apply through contract sub-types such as Resources, Resource interfaces, or other types
+declared in the contract standard. Standards can define and limit behaviour and/or set conditions which
+implementations of the standard cannot violate.
+
+Detailed information about available standards and other core contracts can be found in [Core contracts](flow#core-contracts).
```

Comment:
There's nothing about standards in the core contracts, those are the contracts that govern the chain

Reply:
It's all combined in this page https://developers.flow.com/flow/core-contracts

Reply:
Fixed it